Assets/MRTK/SDK/Experimental/SpatialAwareness/SurfaceMeshesToPlanes.cs

```diff
@@ -0,0 +1,484 @@
+        /// </summary>
+        public float CeilingBuffer
+        {
+            get => ceilingBuffer;
+            set => ceilingBuffer = value;
+        }
+
+        [SerializeField]
+        [Tooltip("Thickness of rendered plane objects")]
+        [Range(0.0f, 1.0f)]
+        private float planeThickness = 0.01f;
+
+        /// <summary>
+        /// Thickness of rendered plane objects
+        /// </summary>
+        public float PlaneThickness
+        {
+            get => planeThickness;
+            set => planeThickness = value;
+        }
+
+        [HideInInspector]
+        private SpatialAwarenessSurfaceTypes drawPlanesMask =
+            (SpatialAwarenessSurfaceTypes.Wall | SpatialAwarenessSurfaceTypes.Floor | SpatialAwarenessSurfaceTypes.Ceiling | SpatialAwarenessSurfaceTypes.Platform);
+
+        /// <summary>
+        /// Determines which plane types should be rendered
+        /// </summary>
+        public SpatialAwarenessSurfaceTypes DrawPlanesMask
+        {
+            get => drawPlanesMask;
+            set => drawPlanesMask = value;
+        }
+
+        [HideInInspector]
+        private SpatialAwarenessSurfaceTypes destroyPlanesMask = SpatialAwarenessSurfaceTypes.Unknown;
+
+        /// <summary>
+        /// Determines which plane types should be discarded.
+        /// Use this when the spatial mapping mesh is a better fit for the surface (ex: round tables).
+        /// </summary>
+        public SpatialAwarenessSurfaceTypes DestroyPlanesMask
+        {
+            get => destroyPlanesMask;
+            set => destroyPlanesMask = value;
+        }
+
+#if PLANE_FINDING_PRESENT
+        /// <summary>
+        /// Delegate which is called when the MakePlanesCompleted event is triggered.
+        /// </summary>
+        public delegate void EventHandler(object source, EventArgs args);
+
+        /// <summary>
+        /// EventHandler which is triggered when the MakePlanesRoutine is finished.
+        /// </summary>
+        public event EventHandler MakePlanesComplete;
+#endif // PLANE_FINDING_PRESENT
+
+        /// <summary>
+        /// Indicates whether or not the project contains the required components for SurfaceMeshesToPlanes
+        /// to enable plane creation.
+        /// </summary>
+        /// <remarks>
+        /// For SurfaceMeshesToPlanes to create planes, the Mixed Reality Toolkit Plane Finding package
+        /// must be imported.
+        /// </remarks>
+        public static bool CanCreatePlanes =>
+#if PLANE_FINDING_PRESENT
+            true;
+#else
+            false;
+#endif // PLANE_FINDING_PRESENT
+
+        #endregion
+
+        #region Private Properties
+
+<<<<<<< HEAD
+        private float floorYPosition;
+        private float ceilingYPosition;
+        private List<SpatialAwarenessPlanarObject> activePlanes;
+=======
+        private List<PlaneWithType> activePlanes;
+>>>>>>> f1c4669f8bc72ac3f1797896faabdf7c876a7712
```

Comment:
merge markers here. you can remove 

```c#
private float floorYPosition;
private float ceilingYPosition;
```

since they're now down at line 200

Reply:
thx. good catch!